hack/runtime-migrator/README-rr.md

```diff
@@ -1 +1,178 @@
-# Runtime Restorer
\ No newline at end of file
+# Runtime Restore
+
+The `runtime-restore` application
+1. connects to a Gardener project, and KCP cluster
+2. retrieves all existing shoot specifications
+3. for each runtime on input list:
+   1. gets Shoot, Cluster Role Bindings and OpenIDConnect resources from the backup
+   2. updates the Runtime CR to label the runtime as controlled by Provisioner to prevent KIM from modifying the shoot and the runtime resources
```

Comment:
```suggestion
   2.  To prevent KIM from modifying the shoot and the runtime resources, update the Runtime CR to label the runtime as controlled by Provisioner.
```